src/backend/InvenTree/company/api.py

```diff
@@ -345,17 +375,9 @@ def get_queryset(self, *args, **kwargs):
     def get_serializer(self, *args, **kwargs):
         """Return serializer instance for this endpoint."""
         # Do we wish to include extra detail?
-        try:
-            params = self.request.query_params
-            kwargs['part_detail'] = str2bool(params.get('part_detail', None))
-            kwargs['supplier_detail'] = str2bool(params.get('supplier_detail', True))
-            kwargs['manufacturer_detail'] = str2bool(
-                params.get('manufacturer_detail', None)
-            )
-            kwargs['pretty'] = str2bool(params.get('pretty', None))
-        except AttributeError:
-            pass
+        params = self.request.query_params
 
+        kwargs.update(self.output_options.format_params(params))
         kwargs['context'] = self.get_serializer_context()
 
         return super().get_serializer(*args, **kwargs)
```

Comment:
Could we move this section to generalized mixin that we can add to views? Not sure if we need the context part, but the rest seems like it would make sense in a mixin, maybe even as part of the general views.